src/main/webapp/mocks/server-response.js

```diff
@@ -0,0 +1,18 @@
+export const mockServerResponse = {
+    links: {
+        create: '/servicesNS/nobody/Splunk_TA_aws/splunk_ta_aws_aws_billing_cur/_new',
+    },
+    origin: 'https://10.202.12.58:8000/servicesNS/nobody/Splunk_TA_aws/splunk_ta_aws_aws_billing_cur',
```

Comment:
Does this IP has any particular use? Anyway - I would avoid any specific IPs in our code - the fact that we reveal existing (?)  IP in Public GitHub repo is a good reason to remove it.

Reply:
You are absolutely right. Removed